dataprofiler/profilers/utils.py

```diff
@@ -91,6 +91,19 @@ def _combine_unique_sets(a: list | set, b: list | set) -> list:
     return list(combined_list)
 
 
+def get_random_number_generator() -> np.random._generator.Generator:
```

Comment:
Should we move this elsewhere in the repo so we don't have to duplicate it?

Reply:
Open to your suggestions.
@taylorfturner do you have thoughts as well?

Reply:
Yeah, I'd say it should be one generator across the library. My one counter to that though is that we have a `utils` specific to `data_reader` and one for `profiles` -- not to say this isn't a case we shouldn't create a global `utils.py` though. 

I don't think it makes intuitive sense to reference the profile utils for `data_reader.data_utils.py`. 

I definitely prefer the idea of a global `utils.py` though